### PR TITLE
Update Card component props

### PR DESCRIPTION
**Pull request by jufglanville**
- Jira Ticket: [MEN-3](https://jufglanvlle.atlassian.net/browse/MEN-3?atlOrigin=eyJpIjoiMWI2ZjQxNTRlZjJjNGMzZjlmMjVmOWY5ZDYwZmMxZDciLCJwIjoiaiJ9) 

- Update Card component to accept Image, header and text props

### CODE REVIEW

File: src/components/SummaryCard.tsx [hunk 1]
```diff
@@ -2,18 +2,20 @@ import React from 'react';
...
-import hero from '../assets/illustration-hero.svg';
 
-const SummaryCard = () => {
+interface Props {
+  image: string;
+  heading: string;
+  text: string;
+}
+
+const SummaryCard = ({ image, heading, text }: Props) => {
```
Comment (JakeStaTeresa) [inline src/components/SummaryCard.tsx]: I think we should call this PriceCard

File: src/App.tsx [hunk 2]
```diff
@@ -3,11 +3,17 @@ import styled from 'styled-components';
 import SummaryCard from './components/SummaryCard';
 
 import background from './assets/pattern-background-desktop.svg';
+import hero from './assets/illustration-hero.svg';
 
 const App = () => {
```
Comment (JakeStaTeresa) [inline src/App.tsx:19]: The state for the selected PricePlan should be here

**Review by JakeStaTeresa — CHANGES_REQUESTED** [review]

File: src/components/SummaryCard.tsx (hunk 1 above)
Comment (jufglanville) [reply src/components/SummaryCard.tsx]: This has been updated

File: src/App.tsx (hunk 2 above)
Comment (jufglanville) [reply src/App.tsx:19]: I'm not too sure what to do here, as this is one of the original branches and the logic will change quite a lot in MEN-5 branch which is to do with the creation of the PricePlan component itself. 
What I have done is simply added a `pricePlanObj` to `App.tsx` and have passed this as a prop to the `PriceCard` component.

Comment (JakeStaTeresa) [reply src/App.tsx:19]: In this case, we are introducing a dependency between the 2 branches. We will have to wait for that branch to get fixed. Before we can progress on this one.

**Pull request merged**